locations/dict_parser.py

```diff
@@ -183,10 +183,10 @@ def get_variations(key):
         flatcase = key.lower().replace("-", "")
         results.add(flatcase)
 
-        FLATCASEUPPER = flatcase.upper()
```

Comment:
Can you make this file follow the casing rules and add comments with what the casing names mean instead of using noqa?